dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+                            instanceName,
+                            _settings.TableName,
+                            new Table
+                            {
+                                Granularity = Table.Types.TimestampGranularity.Millis,
+                                ColumnFamilies =
+                                {
+                                    { _settings.ColumnFamily, new ColumnFamily { GcRule = new GcRule { MaxNumVersions = 1 } } }
+                                }
+                            });
+
+                        Console.WriteLine($"Table {_settings.TableName} created succsessfully\n");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Table {_settings.TableName} already exist");
+                    }
+
+                    #endregion
+
+                    #region Insert multiple rows in bulk using MutateRows
+
+                    /* Each row has a unique row key.
+                                       
+                       Note: This example uses sequential numeric IDs for simplicity, but
+                       this can result in poor performance in a production application.
+                       Since rows are stored in sorted order by key, sequential keys can
+                       result in poor distribution of operations across nodes.
+                     
+                       For more information about how to design a Bigtable schema for the
+                       best performance, see the documentation:
+                      
+                       https://cloud.google.com/bigtable/docs/schema-design */
+
+                    Console.WriteLine($"Writing some greetings to the table {_settings.TableName}");
+
+                    //string[] greetings = new string[] { "Hello World!", "Hellow Bigtable!", "Hellow C#!" };
+
+                    BigtableClient bigtableClient = BigtableClient.Create();
+
+                    //building a MutateRowsRequest (contains table name and collection of entries)
+                    MutateRowsRequest request = new MutateRowsRequest
+                    {
+                        TableNameAsTableName = _tableClient
+                    };
+
+                    for (int i = 0; i < _settings.Greetings.Length; i++)
+                    {
+                        string rowKey = String.Format("{0}{1}", _settings.RowKeyPrefix, i);
+
+                        //Building an entry for every greeting (contains of row key and collection of mutations)
+                        MutateRowsRequest.Types.Entry _entry = new Google.Cloud.Bigtable.V2.MutateRowsRequest.Types.Entry();
+                        _entry.Mutations.Add(Mutations.SetCell(_settings.ColumnFamily, _settings.ColumnName, new BigtableByteString(_settings.Greetings[i])));
+                        _entry.RowKey = ByteString.CopyFromUtf8(rowKey);
+
+                        //adding entry to request
```

Comment:
This comment repeats the code entirely, it's similar to:

```c++
i++;   // increment i
```

As such, it can be removed without loss of information. If there's some higher-level purpose to this next line of code, it should be documented.